libcaf_net/caf/net/http/request_header.hpp

```diff
@@ -151,6 +148,15 @@ class CAF_NET_EXPORT request_header {
   std::pair<status, std::string_view> parse(std::string_view raw);
 
 private:
+  // Finds a field by using case insensitive key comparison. Returns the
+  // iterator pointing to the found field, or to the end if no field is found.
+  std::vector<std::pair<std::string_view, std::string_view>>::const_iterator
+  find_by_key_icase(std::string_view key) const noexcept {
```

Comment:
```suggestion
  // An unsorted "map" type for storing key/value pairs.
  using fields_map = std::vector<std::pair<std::string_view, std::string_view>>;

  // Finds a field by using case insensitive key comparison. Returns the
  // iterator pointing to the found field, or to the end if no field is found.
  fields_map::const_iterator
  find_by_key_icase(std::string_view key) const noexcept {
```

(and for the `fields_` member)